app/components/ui/molecules/SearchBox.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import { escapeRegExp } from 'lodash'
+import theme from '@elifesciences/elife-theme'
+
+import SearchBox from './SearchBox'
+
+const searchOptions = [
+  { value: 'first option' },
+  { value: 'final option' },
+  { value: 'second option' },
+]
+
+function filterFunction(options, searchValue, field) {
+  if (!searchValue) return options
+
+  const inputValue = searchValue.trim().toLowerCase()
+  if (!inputValue) return options
+
+  return options.filter(option =>
+    option[field].toLowerCase().includes(inputValue),
+  )
+}
+
+function getMatchIndex(inputValue, option) {
+  const re = new RegExp(escapeRegExp(inputValue))
+  const match = re.exec(option.toLowerCase())
+  if (match) return match.index
+  return -1
+}
```

Comment:
`filterFunction` and `getMatchIndex` are mocks, and it would be useful to mark them as such. Having them declared as above, with normal function names, gives the impression that they're helper functions to use within the test.

I recommend doing either of the following approaches:
```
function mockFilterFunction() {...}
```

```
let mockFilterFunction;

beforeEach(() => {
  mockFilterFunction = jest.fn();
});
...
it('shows the correct suggestions when "option" is typed in', () => {
  mockFilterFunction.returns(['expected', 'values'])
  const wrapper = mount(<SearchBox filterFunction={mockFilterFunction} />)
  expect(wrapper.instance().state.suggestions).toEqual(['expected', 'values'])
})
```

Reply:
Makes sense. I'll just go with mock function implementation https://jestjs.io/docs/en/mock-functions#mock-implementations